index.js

```diff
@@ -1,15 +1,21 @@
+require('newrelic');
+
 const express = require('express');
 const morgan = require('morgan');
```

Comment:
remove morgan, it is middleware that slows down your server. should only be used for dev

Reply:
(remember that console.logs are very slow operations)

Reply:
Excellent point! Thank you. Resolved